.github/workflows/Ci.yml

```diff
@@ -0,0 +1,107 @@
+name: "CI: Run tests"
+
+# GitHub Actions triggers based on the PR's and each commits on develop and master
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+  push:
+    branches: [develop, master]
+
+jobs:
+  build:
+    defaults:
+      run:
+        shell: bash
+        working-directory: "/opt/carma/"
+
+    # Specify the runner environment
+    runs-on: ubuntu-latest
+
+    # Set up a Docker container for the job
+    container:
+      image: usdotfhwastoldev/autoware.ai:develop
+      env:
+        INIT_ENV: "/home/carma/.base-image/init-env.sh"
+        ROS_2_ENV: "/opt/ros/foxy/setup.bash"
+        TERM: xterm
+      options: "--user root"
+
+    steps:
+    - name: Checkout ${{ github.event.repository.name }}
+      # Check out the repository code
+      uses: actions/checkout@v3.3.0
+      with:
+        fetch-depth: 0
+        path: src/${{ github.event.repository.name }}
+
+    - name: Install Dlib  # Install the Dlib library

```

Comment:
Installation of Dlib can be removed. This is handled in the `multiple_obect_tracking` repo's scripts/install_dependencies.sh script, which is called during carma-platform's build process at https://github.com/usdot-fhwa-stol/carma-platform/blob/3707df6886e4fa7bf0e80dbcbc0356e0e2d78377/docker/install.sh#L37

Reply:
On checking with @adamlm the install_dependencies script needs to be run here to get all the required dependencies for the package